vm-virtio/src/console.rs

```diff
@@ -375,12 +477,16 @@ impl VirtioDevice for Console {
                 input_queue_evt: queue_evts.remove(0),
                 output_queue_evt: queue_evts.remove(0),
                 input_evt: self.input.input_evt.try_clone().unwrap(),
+                config_evt: self.input.config_evt.try_clone().unwrap(),
                 kill_evt,
             };
 
             let worker_result = thread::Builder::new()
                 .name("virtio_console".to_string())
-                .spawn(move || handler.run());
+                .spawn(move || {
+                    (cb)(&VirtioInterruptType::Config, None).unwrap();
```

Comment:
This should be invoked before to spawn the new thread, avoiding the need for cloning.

Reply:
Done.